spec/client_spec.rb

```diff
@@ -1,6 +1,10 @@
 require "spec_helper"
 
 describe "client" do
+  let(:pool_nodes) {@redis.instance_variable_get("@pool").nodes}
+  let(:pool_hosts) {pool_nodes.map{|n| n.host_hash[:host]}}
+  let(:pool_ports) {pool_nodes.map{|n| n.host_hash[:port]}}
```

Comment:
I noticed that these types of checks were being made in multiple places in the test suite, so I've hoisted them all up as global helpers.